index.bs

```diff
@@ -2301,7 +2311,10 @@ The following [=simple exceptions=] can be raised:
 
 </dl>
 
-### Use an Existing Credential to Make an Assertion - PublicKeyCredential's `[[Get]](options)` Method ### {#sctn-getAssertion}
+### Use an Existing Credential to Make an Assertion - PublicKeyCredential's <code><dfn for="PublicKeyCredential" method>[DISCOVER-METHOD-DEF]</dfn></code> Internal Method ### {#sctn-getAssertion}
```

Comment:
The proposed text is more accurate, but now the discussion of why `[[CollectFromExternalSource]]` has a default implementation and `[[DiscoverFromExternalSource]]` does the heavy lifting (essentially, how webauthn hooks into `navigator.credentials.get`) is more spread out. Some of the discussion on the following paras is not directly related to `[[DiscoverFromExternalSource]]` (e.g. the conditional mediation bits).

I would prefer if this header remained, perhaps named

```suggestion
### Use an Existing Credential to Make an Assertion ### {#sctn-getAssertion}
```

And then bring back the heading on L2336 to cover the discussion of `[[DiscoverFromExternalSource]]` as a subsection.

Reply:
Ok, I'll take another pass at this.

> Some of the discussion on the following paras is not directly related to `[[DiscoverFromExternalSource]]` (e.g. the conditional mediation bits).

Ah, good point!